multimodal/tests/unittests/predictor/test_predictor.py

```diff
@@ -621,3 +624,35 @@ def test_modifying_duplicate_model_names():
     for per_modality_processors in teacher_predictor._data_processors.values():
         for per_processor in per_modality_processors:
             assert per_processor.prefix in teacher_predictor._config.model.names
+
+
+def test_image_bytearray():
+    download_dir = "./"
+    train_data_1, test_data_1 = shopee_dataset(download_dir=download_dir)
+    train_data_2, test_data_2 = shopee_dataset(download_dir=download_dir, is_bytearray=True)
+    predictor_1 = MultiModalPredictor(
+        label="label",
+    )
+    predictor_2 = MultiModalPredictor(
+        label="label",
+    )
+    model_names = ["timm_image"]
+    hyperparameters = {
+        "optimization.max_epochs": 2,
+        "model.names": model_names,
+        "model.timm_image.checkpoint_name": "swin_tiny_patch4_window7_224",
```

Comment:
Sorry for not mentioning before, since this is a new input format, we want to test it as comprehensive as possible. So can you (1) add a CNN model like `mobilenetv3_large100` for testing (2) test `.predict()` and `.predict_proba()` functions as well in the unit test? Thank you.

Reply:
Updated
(1) use CNN model https://github.com/autogluon/autogluon/blob/909b7c3217121fa6a77ad6f23c5b235a77a51d96/multimodal/tests/unittests/predictor/test_predictor.py#L125
(2) test `.predict()` and `.predict_proba()`: https://github.com/autogluon/autogluon/blob/909b7c3217121fa6a77ad6f23c5b235a77a51d96/multimodal/tests/unittests/predictor/test_predictor.py#L663-L664